### ISSUE TITLE
Make pip package

### ISSUE THREAD

**Issue by pamelafox** [kjaymiller/Az-Queue-Tweeter#14]
I think it'd be good to have a structure like:

- `queuetweeter`
    - `__init__.py` (with QueueTweeter class)
    - `storage.py`
    - `twitter.py`

Then people could use like:

```
from queuetweeter import QueueTweeter, storage, twitter

qt = QueueTweeter(storage.Auth(..), twitter.Auth(..))
```

It's probably best if you do the packaging so its under your account.

**Comment by kjaymiller** [comment]
Sounds good. I'll get that wrapped up and get a beta release of this out the door.

**Comment by kjaymiller** [comment]
resolved in #15
